components/authz-service/projectassignment/projectassignment_test.go

```diff
@@ -0,0 +1,45 @@
+package projectassignment
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCalculateProjectDiff(t *testing.T) {
+	cases := []struct {
+		desc string
+		f    func(*testing.T)
+	}{
```

Comment:
Please add tests for:
- no projects added or removed for a _non-empty_ project list
- no projects at all (which is, of course, equivalent to: no projects added or removed for an _empty_ project list)